chapter10/ajd/exercise2.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Benchmark qw( :all );
+
+sub naive {
+  chdir;
+  my @sorted = sort { -s $a <=> -s $b } glob '*';
+  # Explicit return value to stop the array being returned
+  # Turns out Benchmark goes off and tests all the scripts that this returns
```

Comment:
Do you have a link for this? I couldn't find it listed in the CPAN docs, and I don't _think_ mine exhibited this behaviour despite returning the array of values.